metrics/configs/istio-flakes.yaml

```diff
@@ -0,0 +1,31 @@
+# Istio has identical names for pre/post submits and
+# the version field in the job results are alwasys `unknown`
+# so the canonical flakiness metrics config does not apply to
+# istio. This config is used by istio only and generates weekly
+# flakiness summery given the constraints aforementioned.
+metric: istio-job-flakes
+description: Computes flakiness for all jobs.
+query: |
+  SELECT
+    job,
+    sum(runs) runs,
+    sum(passed) passed,
+    if(passed == 0, 1, sum(passed)/sum(runs)) consistency,
+    max(stamp) stamp,
+  FROM (
+    SELECT /* all runs of any job for the past week, noting whether it passed */
+      job,
+      regexp_extract(metadata.value, r'master:([a-f0-9]+)"') commit, /* extract the master commit sha */
+      date(started) stamp,
+      sum(if(result=='SUCCESS',1,0)) passed,
+      count(result) runs,
+    FROM [k8s-gubernator:build.all]
```

Comment:
Since you're only interested in the past week, please use build.week.

Reply:
I was think something like a 7-day sliding window. But now I think about this, daily results are probably more responsive to changes in data, so I update this to daily. PTAL

Reply:
Querying the build.all table when all you want is the past day is inefficient on BigQuery, which doesn't have indexes like normal SQL databases. If you change this query to use the `[k8s-gubernator:build.day]` table, it will have the same results and have to query over less data.

Reply:
Got it. Thanks for the insights!